### DISCUSSION TITLE
Unable to use other configurations than Debug and Release in MAUI

### DISCUSSION THREAD

**Discussion by gandhimonil** [dotnet/maui#14655] [Q&A] [answered]
I want more configurations like UAT, Pre-Prod but I can only run Debug.

I have added more configurations from the project settings:

![Screenshot 2023-04-19 at 4 30 28 PM](https://user-images.githubusercontent.com/22075514/233056325-a5981689-4a43-4f06-87d5-5b697a392b25.png)

If I run other configurations, I am getting `Selected exexution mode is not supported for .Net Projects` error.

**Comment by FM1973** [comment]
Hello.

I´m using conditions in the project file like this:

`<PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='DebugStaging|net7.0-ios|AnyCPU'">`

just google it. You will find samples.

**Reply by gandhimonil** [reply]
I tried this already. It is working fine in Visual Studio 2022(Windows) but getting error in Visual Studio for MAC 2022.

**Comment by gandhimonil** [comment, marked as answer]
I found the solution. So basically other configuration works in **Visual Studio 2022(Windows)** but gets an error in **Visual Studio for MAC 2022** [Issue Link](https://github.com/dotnet/maui/issues/8894#issuecomment-1210331944)

Until **Visual Studio For MAC** solves this issue we can run different configurations using the terminal using the commands link:

**Command:** `dotnet build -t:Run -f net7.0-ios -p:_DeviceName=:v2:udid=MY_SPECIFIC_UDID -c <Configuration>`

**ex:** `dotnet build -t:Run -f net7.0-ios -p:_DeviceName=:v2:udid=2B4C2D4C-18E2-4C5F-8A38-51CE611A1899 -c UAT`

we can skip the DeviceName part. Which will run the app in the default simulator.
**Command:** `dotnet build -t:Run -f net7.0-ios -c UAT`
